airflow/hooks/http_hook.py

```diff
@@ -142,8 +144,13 @@ def check_response(self, response):
         try:
             response.raise_for_status()
         except requests.exceptions.HTTPError:
-            self.log.error("HTTP error: %s", response.reason)
-            self.log.error(response.text)
+            # mark the exception if colored log is supported
+            if conf.getboolean('core', 'colored_ui_logs'):
+                self.log.error(Fore.WHITE + Back.RED + "HTTP error: %s", response.reason + Style.RESET_ALL)
+                self.log.error(Fore.WHITE + Back.RED + response.text + Style.RESET_ALL)
```

Comment:
If I am not mistaken the log will be colored two times? Once by color log formatter, the second time by adding codes here?

Reply:
I don't understand. where is this color log formatter?

Reply:
You are talking about this file?
`airflow/utils/log/colored_log.py 
`

Reply:
I removed this change (and the colorama package)